bolt-simple-taproot.md

```diff
@@ -296,6 +318,14 @@ The `Sign` is used to generate a valid partial signature, with the
 Once all partial signatures are known, the `PartialSigAgg` algorithm is used to
 combine the partial signature into a final valid BIP 340 schnorr signature.
 
+##### Partial Signature Encoding
+
+A `musig2` partial signature is simply the encoding of the `s` value which is
+32 byte element modulo the order of the group. Throughout this specification,
+in order to be able to re-use the existing 64-byte signature field, partial
+signatures are also encoded with 64 bytes by also encoding the x-coordinate of
+the _final_ nonce (`R1+b*R2`).
```

Comment:
(the spec defines this as the "effective" nonce, per https://github.com/jonasnick/bips/issues/69#issuecomment-1244322742) :)

Reply:
I stand corrected, actually, so nvm lol